### DISCUSSION TITLE
Unable to start on Mac OS Sonoma.

### DISCUSSION THREAD

**Discussion by cruxim** [audacity/audacity#5614] [Q&A]
### Bug description

After upgrading to macOS Sonoma, No matter which version, no matter how to reinstall, it will display:

[Audacity failed to start.

Audacity IPC server failed to initialize.

This may be due to insufficient resources and may require a restart.]

and restart didn't work.

<img width="260" alt="image" src="https://github.com/audacity/audacity/assets/45716155/45543275-193c-4eff-8f4c-51efd0836460">
![WeChatc50dc06e3d4d6d7ac1cfa09c8b1a68c0](https://github.com/audacity/audacity/assets/45716155/79967201-23f2-457f-bbf8-23029c5c8736)

### Steps to reproduce

just open it

### Expected behavior

the software should been open

### Audacity Version

current master / alpha / nightly build

### Operating system

macOS

**Comment by dozzzzer** [comment]
@cruxim If anything, I've found this [forum thread](https://forum.audacityteam.org/t/cannot-open-audacity-error-message-ipc-server-failed-to-initialize/63406/2) describing the same issue. Do you have enough free space on your disk?

**Comment by cruxim** [comment]
> @cruxim If anything, I've found this [forum thread](https://forum.audacityteam.org/t/cannot-open-audacity-error-message-ipc-server-failed-to-initialize/63406/2) describing the same issue. Do you have enough free space on your disk?

Thank you! But unfortunately, my hard drive still has 300GB of capacity, so it shouldn't be a problem in this aspect. I have also read the forum post and tried uninstalling and reinstalling completely, but it still didn't work~

**Comment by LWinterberg** [comment]
macOS specifically had  some trouble before 3.4.0. 

Additionally, https://support.audacityteam.org/troubleshooting/error-codes#the-audacity-ipc-server-failed-to-initialize (and the point below that) may be useful here

**Comment by estefan3112** [comment]
Latest version 3.4.2 works here on Sonoma without issues.
